### ISSUE TITLE
Weekly Community Check-In #114 - Self Development and its Importance

### ISSUE THREAD

**Issue by unnati914** [publiclab/plots2#10045]
Hi everybody! 👏

We all at Public Lab 🎈 - learn, grow, work, brainstorm ideas, contribute together so why not share about our weekly goals and the awesome work we have done at Public Lab with each other, so we can support and collaborate with each other better. We have a Community Check-In each week, where every community member can share something about their work from the past week and about their current week's goal 🎯 . You are also welcome to share fun-fact 😄 , new ideas 💡 , your learning goals ☑️.

We believe in collaborative efforts to support our community. We are running a learning platform which helps a newcomer to become a leader of tomorrow. 💯

## Title : Self development and its Importance

Self development is taking steps to better yourself, such as by learning new skills or overcoming bad habits. An example of self development is taking courses at the university to learn new skills and interesting things. ... Efforts toward self-fulfillment, either through formal study programs or on one's own.
This week share about your development in open source, how you started, what motivated you to start it and Why do you think its important for self development 💯 

## This week's focus Issues

These are issues that are of high priority and requests from our
large community that use the software. 

https://github.com/publiclab/plots2/labels/high-priority

We will very much appreciate any help on these issues. Any contributor who is looking for issues to work on please consider taking a look at these.
We will add a thank you message here in next week's check-in if you contribute to any of these. Thanks in advance 😄

### Thank you note to focus issues contributors
 Thank the contributors who worked on focus issues last week 💯
@jywarren , @17sushmita , @imajit , @Manasa2850 , @TildaDares , @vacom13 , @sorutifokkusu , @aummishra 🥇 

## Next Check-in
If you would like to open the next check-in..Leave a comment below. We are happy to help if its your first time

**Thanks everyone for making Public Lab awesome** :heart:  
Have a great week ahead all :balloon:

**Comment by noi5e** [comment]
@unnati914 Thanks so much for making this check-in!!

I started working in open source as an Outreachy intern. It's been a very educational experience. I've learned a lot more about programming through working on open source than working on my own projects and tutorials.

I find that doing programming work helps my self development too. It develops the problem-solving part of my brain, and helps me to not get too discouraged and overwhelmed by other areas of my life. I think that programming teaches you how to persevere and solve problems through patience.

**Comment by Tlazypanda** [comment]
Moved to #10085

**Comment by jywarren** [comment]
@unnati914 I wanted to circle back and share that my intro to open source was using Wordpress -- actually I think I remember using WordPress 1.0??? 😱 

I made WordPress sites and themes and then plugins... for years! before getting deeper into open source coding through Processing.org. For a long time I didn't feel like a programmer exactly, more like a designer who could program. But gradually i built up experience and skills and met people and so forth. It's been a journey! And most remarkable to me is that there have been moments where I've had to re-learn fundamentals -- to restart -- as either technologies changed (like the rise of NPM and modular JavaScript) or as collaborative practices evolved -- like all the unique work we do here at Public Lab. 

Thanks for a great question!
